docs/topics/jvm/jvm-test-using-junit.md

```diff
@@ -0,0 +1,149 @@
+
+2. Open the `build.gradle(.kts)` file and add the following dependencies to the Gradle configuration to work with `kotlin.test` and `JUnit`:
+
+   <tabs>
+
+   ```groovy
+   dependencies {
+       // Other dependencies.
+       testImplementation 'org.jetbrains.kotlin:kotlin-test-junit5'
+       testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
+       testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.6.0'
+   }
+   ```
+
+   ```kotlin
+   dependencies {
+       // Other dependencies.
+       testImplementation(kotlin("test-junit5"))
+       testImplementation("org.junit.jupiter:junit-jupiter-api:5.6.0")
+       testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.6.0")
+   }
+   ```
+
+   </tabs>
+
+3. Ensure, that **Project Wizard** automatically created the `test` task in the `build.gradle(.kts)` file:
+
+   <tabs>
+
+   ```groovy
+   test {
+       useJUnitPlatform()
+   }
+   ```
+
+   ```kotlin
+   tasks.test {
+       useJUnit()
+   }
+   ```
+
+   </tabs>
+
+## Add code to test
+
+1. Open the `main.kt` file in `src/main/kotlin`.
+
+   The `src` directory contains Kotlin source files and resources. The `main.kt` file contains sample code that will print `Hello World!`.
+
+2. Create the `Sample` class with the `sum()` function that sums two integers:
+
+```kotlin
+class Sample() {
+
+    fun sum(a: Int, b: Int): Int {
+        return a + b
+    }
+}
+```
+
+## Create a test
+
+1. In IntelliJ IDEA, select **Code** \| **Generate** \| **Test...** of the `Sample` class.
+
+   ![Create a test](create-test.png)
+
+2. Specify the name of the test class. For example, `SampleTest`.
+
+   IDEA creates the `SampleTest.kt` file in the `test` directory. It contains Kotlin test source files and resources.
+
+   > You can also create a `*.kt` file for test in `src/test/kotlin` manually.
+   >
+   {type="note"}
+
+2. Add the test code that for the `sum()` function in `SampleTest.kt`:
+   
+   * Define the test `testSum()` function using the [@Test annotation](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/-test/index.html).
+   * Check that the `sum()` function returns the value that is equal to the expected one using the [assertEquals()](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/-test/assert-equals.html) function.
+
+   ```kotlin
+   import kotlin.test.Test
+   import kotlin.test.assertEquals
+   
+   internal class SampleTest {
+   
+       private val classForTesting: Sample = Sample()
+   
+       @Test
+       fun testSum() {
+           val expected = 42
+           assertEquals(expected, classForTesting.sum(40, 2))
+       }
+   }
+   ```
+
+## Run a test
+
+1. Run the test using the gutter icon.
+
+   ![Run the test](run-test.png)
+
+   > You can also run all project tests via the command-line interface using the `./gradlew test` command.
+   > 
+   {type="note"} 
+
+2. Check the result of the test run in the **Run** tool window:
+
+   ![Check the reult](check-the-result.png)
```

Comment:
Typo `reult`. Also, I'd make more meaningful alt text - the image with the failed text has almost the same. Smth with `success` here and with `fail` below.